compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/programfile/PackageInfoWriter.java

```diff
@@ -582,11 +582,14 @@ private static void writeAttributeInfo(DataOutputStream dataOutStream,
                 attrDataOutStream.writeShort(docAttrInfo.paramDocInfoList.size());
                 for (ParameterDocumentInfo paramDocInfo : docAttrInfo.paramDocInfoList) {
                     attrDataOutStream.writeInt(paramDocInfo.nameCPIndex);
-                    //TODO remove below line ASAP, adding dummy value as we can't change binary file right now
-                    attrDataOutStream.writeInt(1);
-                    attrDataOutStream.writeInt(paramDocInfo.paramKindCPIndex);
                     attrDataOutStream.writeInt(paramDocInfo.descriptionCPIndex);
                 }
+                if (docAttrInfo.returnParameterDescriptionCPIndex != -1) {
```

Comment:
shall we write just cp index and differentiate at reader side?

Reply:
We have discussed offline and decided to keep this as it is since the boolean value size which will be written to the file is smaller than an integer value.